### DISCUSSION TITLE
Who is "volth" and why do they appear as the user on archive.ph?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [community/community#58164] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm not sure if this is the right place to ask, but this has been bugging me for a long time and I don't know anywhere better to ask. If you look at any of the latest pages saved by archive.ph, such as <https://archive.is/lX3Ej>, you can see the page logged in as "volth". You can even see [their notifications](https://archive.is/Y7z4E). However, "volth" [is not a real user](https://github.com/volth) since [~2021](https://archive.is/https://github.com/volth), but the archiver is still logged in as them. Additionally, attempting to archive settings/profile would present you with a [login screen](https://archive.is/RlY5r), yet on other pages the user still isn't signed out. Is there a possible explanation for this?

**Comment by Et7f3** [comment]
It seems to be a user that deleted this account ? I also noticed his disparition.
